docs/deployment.md

```diff
@@ -88,3 +88,23 @@ To deploy to production server:
   ```shell
   sudo systemctl start docker-osmaxx.service
   ```
+
+## Deploying a Hotfix
+
+Follow the above steps until 5a.
+
+Replace step 5b with:
+
+5. 
```

Comment:
You could use HTML in MarkDown and [achieve the decided effect with `ol`'s `start` attribute](http://www.echoecho.com/htmllists02.htm), but as markdown inside a tag isn't interpreted, this would be a PITA. You'd have to write the whole `ol` content in HTML.